spec/factories/info_requests.rb

```diff
@@ -6,34 +6,34 @@
     public_body
     user
 
-    after_create do |info_request, evaluator|
-      FactoryGirl.create(:initial_request, :info_request => info_request)
+    after(:create) do |info_request, evaluator|
+      create(:initial_request, :info_request_id => info_request.id)
```

Comment:
In fact this may even have been a "let's try something strange to see if it fixes it" test line which got committed by mistake. If I made notes at the time, I appear to have misplaced them.